e2e_tests/tests/experiment/test_noop.py

```diff
@@ -208,7 +205,7 @@ def test_cancel_one_paused_experiment() -> None:
         conf.fixtures_path("no_op"),
         ["--paused"],
     )
-
+    time.sleep(0.25)
```

Comment:
?

Reply:
ok it looks like I can safely remove the added `time.sleep` calls now that I handle a race condition in `wait_for_experiment_state` https://github.com/determined-ai/determined/pull/3539/files#diff-66c234e5c973d0546d1f6713c5b496deca22124eea269b1f8c5a3fb0c1193c81R116